packages/config/config/devices/0x0312/mp31zp.json

```diff
@@ -0,0 +1,169 @@
+{
+	"manufacturer": "Minoston",
+	"manufacturerId": "0x0312",
+	"label": "MP31ZP",
+	"description": "Z-Wave Plug with Energy Monitor",
+	"devices": [
+		{
+			"productType": "0xff00",
+			"productId": "0xff0e",
```

Comment:
@boomam This information is entered in the Z-Wave Alliance DB, but it seems to duplicate the MP21ZP IDs. I'm guessing this is an error in the DB entry.

Can you post the device ID that is shown for your device? In zwavejs2mqtt, you'll find it when you click on the device in the device list. In HA I'm not sure, but the debug information for the node should probably contain it.

Reply:
The rest is cleaned up, so I can merge this when the correct IDs are known.